backend/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/push/parsing/Impact.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SonarLint Core - Server API
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.serverapi.push.parsing;
```

Comment:
I'm not sure this class belongs in the `parsing` folder

Reply:
It is related to the parsing of the RuleSetChangedEvent, so I think it belongs to this package. I don't think it's intended to be a business class. What you could do though is to move this class as a static inner class of the RuleSetChangedEventParser, because it's related to the parsing of this specific event. The parser file would be self-contained

Reply:
Done